icu4c/source/tools/toolutil/udbgutil.cpp

```diff
@@ -697,8 +704,13 @@ UBool KnownIssues::print()
           std::map <  std::string,  std::set <  std::string > > >::iterator i = fTable.begin();
        i != fTable.end();
        i++ ) {
-    char URL[1024];
-    std::cout << '#' << (*i).first << " <" << udbg_knownIssueURLFrom( (*i).first.c_str(), URL ) << ">" << std::endl;
+    const std::string ticketid = (*i).first;
+    std::cout << "[" << ticketid << "] ";
+    if(!ticketid.rfind(ICU_BUG_PREFIX, 0) || !ticketid.rfind(CLDR_BUG_PREFIX, 0)) {
```

Comment:
This looks confusing. Why rfind() not find()? And please use == 0 not negation.
Too bad string does not have starts_with()...